webapp/components/user_settings/user_settings_security.jsx

```diff
@@ -74,6 +55,16 @@ class SecurityTab extends React.Component {
         };
     }
 
+    componentWillMount() {
```

Comment:
cool I thought for some reason that it was the other way around, I'l make the change